sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
```

Comment:
This is a github action for what? Should we use jenkins?

We areso planning to workflows at top level .github/workflows similar to other infra bits, to keep them all in one place. (see related PRs for wheel files.)

Reply:
I can remove this.

TL;DR:
This workflow is supposed to support the line
```
 ![Github Actions Status](https://github.com/apache/beam/workflows/Build/badge.svg)
```
in the README.md.